content/authors/donnex-wafula/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Donnex Wafula
+type: authors
+github: https://github.com/donnexwafula33
+images:
+  - url: /engineering-education/authors/donnex-wafula/avatar.jpg 
+---
+Donnex Wafula is an undergraduate Computer Science student. His interests are web development and machine learning. Donnex has a great passion for developing web applications, technical writing and building machine learning models. He specializes in natural language processing and image classification.
```

Comment:
Please use the format shown in this [documentation](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/UPLOAD_INSTRUCTIONS.md) to fix your author bio